Sources/TokamakCore/Fiber/Element.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  File.swift
+//
+//
+//  Created by Carson Katri on 2/15/22.
+//
+
+/// A reference type that points to a `Renderer`-specific element that has been mounted.
+/// For instance, a DOM node in the `DOMFiberRenderer`.
+public protocol Element: AnyObject {
```

Comment:
Do I understand correctly that this roughly corresponds to the `Target` protocol used in the stack reconciler? Either way, I find the `Element` name by itself too generic for keeping it in the top-level namespace of the module. Maybe something like `FiberElement` could work?

Reply:
Yes, it does correspond to `Target`. `FiberElement` sounds good.